### ISSUE TITLE
Markus: Translating the search interface

### ISSUE THREAD

**Issue by klokan** [klokantech/kartenportal-tickets#82]
We are going to prepare the localization web interface for translators - so that the phrases in the search interface can be easily translated.

The link will appear here - we will need alert when the translation into all languages is finished.

**Comment by klokan** [comment]
We are preparing the online translation system.

**Comment by klokan** [comment]
@zb-markus Please make ASAP the validation of all English texts at http://search.kartenportal.ch/. If you want to make changes in English then now it is the right moment.

Once this is finished we will generate the translation dictionaries and you or your appointed persons can start to work on the translation from English to other languages. The system is ready.

**Comment by zb-markus** [comment]
Some small amendments reported in #58. No other problems in English texts found so far.

Will the translation system allow to print out the texts to be translated as a list, and/or to expor it as an Excel spreadsheet? Will the system allow translators to see all other languages already translated, for example in a spreadsheet with columns for each language?

Will translators be able to see whether there is a limit in the text length or not (for example when the size of buttons requires short words, versus free text length in the Help box)?

**Comment by klokan** [comment]
@zb-markus please provide me with your Google account.

You can create one for your existing email address (at ZB) via https://accounts.google.com/SignUp

 Translation interface is ready with phrases imported.

**Comment by ethbib-susanne** [comment]
Hello Markus, hello Petr,

J can fill in the translations. My account:

maps@library.ethz.ch

Do you need more information?

**Comment by klokan** [comment]
You have both access to the translation toolkit with the phrases for the Search Interface.

German: http://translate.google.com/toolkit/workbench?did=00cewrc00a4sdfp179c0
French: http://translate.google.com/toolkit/workbench?did=004admb00a4sddq037cw
Italian: http://translate.google.com/toolkit/workbench?did=00702ie00a4sdho2bbb4 
Romansh: http://translate.google.com/toolkit/workbench?did=00a6r3400a4soga8uadc
English corrections: http://translate.google.com/toolkit/workbench?did=002jexa00ac7xqprifwg

@zb-markus I created for you the kartenportal@gmail.com account. Details are in the email.
Let me know once you are ready with the translations.

**Comment by klokan** [comment]
Task to be finished after the delivered translations are applied and visible on the production system:
In the Help dialog, the About part, it should read "You can search for maps covering ..." (insert "for").

UPDATE: There is now an "English corrections" translation too for this sort of adjustments...

**Comment by klokan** [comment]
All translations as right now available in the translation system (including the German and English corrections) is applied on the production system now - we can update the system with phrases from Google Translate on request easily.

**Comment by zb-markus** [comment]
OK, translations are in progress ...

**Comment by zb-markus** [comment]
I think we can close the ticket. Technically the system has been tested and accepted. The content is in progress. We will open a separate ticket when an updated language version is ready.
